cloudsmith/data_source_package.go

```diff
@@ -156,7 +156,7 @@ func dataSourcePackage() *schema.Resource {
 				Type:        schema.TypeString,
 				Description: "The path to save the downloaded package",
 				Optional:    true,
-				Default:     "",
+				Default:     "/tmp/",
```

Comment:
what if the user's on windows or some other platform? can this be OS agnostic? maybe https://github.com/golang/go/blob/master/src/os/file.go#L392 ?